internal/notify/channels-notify.go

```diff
@@ -0,0 +1,58 @@
+package notify
+
+import (
+	"context"
+	"errors"
+	"hellper/internal"
+	"hellper/internal/log"
+)
+
+func channelsNotify(ctx context.Context) {
+
+	var (
+		to, msg string
+	)
+
+	logger.Info(ctx, log.Trace(), log.Action("running"))
+
+	if arg.statusFlag == "" {
+		logger.Error(ctx, log.Trace(), log.NewValue("error", errors.New("Must have a status")))
+		return
+	}
+
+	if arg.toFlag != "" {
+		logger.Error(ctx, log.Trace(), log.NewValue("error", errors.New("Forbidden to use the --to option with --type=channels")))
+		return
+	}
+
+	repository := internal.NewRepository(logger)
```

Comment:
In NewRepository we don't have a error treat on [Internal#L39](https://github.com/ResultadosDigitais/hellper/blob/master/internal/internal.go#L39) . Why?

I understand that it may be an error in the internal package and not the cli notify, I just asked for curiosity. This should not impact this PR.

Reply:
The error conditional is here: 
https://github.com/ResultadosDigitais/hellper/blob/7a2b7a35b7a9dba034cce7d00a194fdd30b067ec/internal/model/sql/sql.go#L78

Reply:
:+1: